np.plugin-flow-skeleton/rollup.config.js

```diff
@@ -0,0 +1,25 @@
+// DO NOT EDIT THIS FILE. IT'S FOR THE SCRIPT PACKAGER!
```

Comment:
You no longer need this file if you just use `nom run autowatch` that one command automatically does the compiling for you.

But feel free to keep it if you think it adds value. I feel like letting one script handles everything automatically is simpler to get started with